examples/react-example/src/App.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
```

Comment:
Let's make these TypeScript files, see: https://create-react-app.dev/docs/adding-typescript/

Reply:
Didn't choose that option on purpose: I thought the example should be the most basic JS example to make it accessible for all type of react projects. Or do you think we would have more benefits if we convert this to TS?